ui/src/app.js

```diff
@@ -2136,3 +2144,30 @@ function setReceiptPrint (receiptStatus, smsReceiptStatus) {
       break
   }
 }
+
+function setScreenOptions (opts) {
+  if (opts.rates) {
+    opts.rates.active ? $('#rates-section').show() : $('#rates-section').hide()
+  }
+}
+
+function setRates (allRates, fiat) {
+  const ratesTable = $('.rates-content')
+  const tableHeader = $(`<div class="xs-margin-bottom">
+  <h4 class="js-i18n">Buy</h4>
+  <h4 class="js-i18n">Crypto</h4>
+  <h4 class="js-i18n">Sell</h4>
+</div>`)
+  const coinEntries = []
+  
+  Object.keys(allRates).forEach(it => {
+    coinEntries.push($(`<div class="xs-margin-bottom">
+    <p class="d2 js-i18n">${allRates[it].cashIn}</p>
+    <h4 class="js-i18n">${it}</h4>
+    <p class="d2 js-i18n">${allRates[it].cashOut}</p>
+  </div>`))
+  })
+
+  $('#rates-fiat-currency').text(fiat)
```

Comment:
Ah I think you can ignore my other comment about the fiat. This line changes it right?

Reply:
yes